pkg/api-server/testdata/resources/inspect/dataplanes/_rules/meshgateway_from_global.input.yaml

```diff
@@ -0,0 +1,77 @@
+#/meshes/default/meshgateways/the-gateway/_rules 200
```

Comment:
Looking at this path this test is in the wrong folder. It should likely be there no: https://github.com/kumahq/kuma/tree/513a70e65cae9f47b69435b4594a02029fdc42c2/pkg/api-server/testdata/resources/inspect/meshgateways ?

Reply:
right, moved